extension/diffheader.js

```diff
@@ -109,10 +107,9 @@ window.diffFileHeader = (() => {
 	const setup = () => {
 		$(window).on('scroll.diffheader', () => diffHeaderFilename());
 		const onResize = utils.debounce(() => diffHeaderFilename(true), 200);
-		$(window).on('resize', onResize);
+		$(window).on('resize.diffheader', onResize);
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5233399/16173583/2d44c45a-356b-11e6-973b-6e1294d8eaf7.png)

It's a namespace. It makes it easier to remove the event handler later, without needing a ref to the handler (like `add/removeEventListener`) or needing to blow away _all_ bound events of that type.